src/main/resources/templates/fragments/reusable-components.html

```diff
@@ -26,7 +26,7 @@ <h5 class="card-title">Collection title</h5>
         <div class="card-body text-dark">
             <h5 class="card-title">Start Here</h5>
             <p class="card-text">Click below to add a new collection</p>
-            <a href="#" class="btn btn-primary">Add Collection</a>
+            <a th:href="@{addEditCollection}" class="btn btn-primary">Add Collection</a>
```

Comment:
Added the th:href tag so that we use Thymeleaf to talk to the controller. 

Citation: [https://www.thymeleaf.org/doc/articles/standardurlsyntax.html](https://www.thymeleaf.org/doc/articles/standardurlsyntax.html)